app/views/foreman/job_templates/install_errata.erb

```diff
@@ -6,10 +6,20 @@ description_format: 'Install errata %{errata}'
 feature: katello_errata_install
 provider_type: SSH
 template_inputs:
+- name: Filter Errata
+  description: Filter criteria for Errata to be installed.
+  input_type: user
+  required: false
+- name: Inclusion Type
+  description: Include/Exclude the errata ids specified below.
+  input_type: user
+  required: true
+  options: "Include the errata below\r\nInclude all errata EXCEPT the below"
```

Comment:
I think using words like 'below' here is quite odd because we're describing this input field based on some aspect of the UI..

Reply:
Well this came of a discussion with @jeremylenz . I am ok with other verbiage. Feel free to suggest alternatives.

Reply:
My suggestion would be to remove words like 'below' which relate to the UI.

Reply:
@jturel The problem is that this _does_ show up in the UI and this is our only way to control the form fields that appear there. (see the screenshot in the PR description above) I couldn't think of a wording that is truly context-agnostic but also makes sense and is clear enough to be self explanatory.  Do you have any other wording suggestions?

Reply:
Could we refer to the field name which seems to be 'errata' (should it be Errata IDs instead?) instead of its location on the screen